api-reference/_hidden/dxDataGridColumn/groupCellTemplate.md

```diff
@@ -60,6 +60,103 @@ The group cell's value with applied [format](/api-reference/_hidden/dxDataGridCo
 The group cell's value as it is specified in a data source.
 
 ---
+
+Group cells display the following data:
+
+* The caption and the value of the column by which data is grouped
+
+* [Group summary items](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/summary/groupItems/)
+
+* [groupContinuesMessage](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/grouping/texts/#groupContinuesMessage) and [groupContinuedMessage](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/grouping/texts/#groupContinuedMessage) (if the group is spread across two or more pages)
+
+The following example shows how to display column value only:
```

Comment:
Is there only one column value? Or do you mean:

`The following example shows how to display column values only:`

Reply:
Users can group data by a certain column. Group cell can display the column value by which data is grouped. They can also display other data. The example shows how to only display column values in the group cells.

Reply:
Thank you. Can we use your suggestion?

`The following example shows how to only display column values in group cells.`

Reply:
No because it is clear that column values are displayed _in group cells_ - groupCellTemplate deals with group cells only.

Reply:
Okay, I see. If we're going to use the singular form ("column value"), then we need to add an article:

```suggestion
The following example shows how to display the column value only:
```